log-skeleton/src/components/split.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from '../styles/Split.module.css';
+
+class SplitActivity extends React.Component {
+    render() {
+      return (
+        //probably button will not be the correct type
+        <div>
+            <button className={styles.activityButton}>Split Activity {this.props.value}</button>
+            <button className={styles.activityButton}>Over Activity {this.props.value + 1}</button>
+        </div>
+      );
+    }
+  }
+
+class Splitter extends React.Component {
```

Comment:
Can you use Functional components instead of Class components? It is nowadays the more convenient solution 😄